### PR TITLE
fix(twig): normalize twig templates, pt8 - INNO-1019

### PR DESCRIPTION
**Pull request by yhuard**
Updated:
- component-logo
- component-message
- component-meta

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 8f74f176415324b55e002ec03527c639ddbac290

https://deploy-preview-660--europa-component-library.netlify.com

File: src/generic/generic-component-logo/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Default EC logo.
 
 ```html
-<a href="https://ec.europa.eu" class="logo" title="European Commission">
+<a href="https://ec.europa.eu" class="ecl-logo" title="European Commission">
```
Comment (kalinchernev) [inline src/generic/generic-component-logo/README.md]: :+1:

File: src/generic/generic-component-message/generic-component-message.twig [hunk 2]
```diff
@@ -1,19 +1,67 @@
...
+{# Internal properties #}
 
-  {% if title is defined %}
-  <div class="ecl-message__title">{{ title }}</div>
-  {% endif %}
+{% set _screen_reader_description = screen_reader_description|default('') %}
+{% set _messages = messages|default([]) %}
+{% set _title = title|default('') %}
+{% set _dismiss = dismiss|default(false) %}
+{% set _dismiss_label = dismiss_label|default('Dismiss this alert') %}
```
Comment (kalinchernev) [inline src/generic/generic-component-message/generic-component-message.twig]: :+1:

File: src/generic/generic-component-site-header/generic-component-site-header.twig [hunk 3]
```diff
@@ -68,7 +68,7 @@
 
   <div class="ecl-container ecl-site-header__banner">
     {% include '@ecl/generic-component-logo' with {
-      'to': 'https://ec.europa.eu',
+      'href': 'https://ec.europa.eu',
       'title': 'Home - European Commission',
```
Comment (kalinchernev) [inline src/generic/generic-component-site-header/generic-component-site-header.twig]: Could you please make this configurable from the component's API? (In the same way you've done [here](https://github.com/ec-europa/europa-component-library/pull/660/files#diff-ee408b90955ce3feb819d7fb7440e89bL5))

It's a question for the whole file, as there's also the `'heading': 'Select your language'`

File: src/generic/generic-component-logo/generic-component-logo.twig [hunk 4]
```diff
@@ -1,21 +1,52 @@
...
+  {% if _type == 'below' %}
+    {% set _css_class = _css_class ~ ' ecl-logo--logotypebelow' %}
+  {% elseif _type == 'right' %}
+    {% set _css_class = _css_class ~ ' ecl-logo--logotype' %}
   {% endif %}
 {% endif %}
 
-{% if extra_classes%}
-  {% set _class = _class ~ " " ~ extra_classes%}
+{% if extra_classes is defined and extra_classes is not empty %}
```
Comment (kalinchernev) [inline src/generic/generic-component-logo/generic-component-logo.twig]: Default is missing for this one.

**Review by kalinchernev — CHANGES_REQUESTED** [review]
One default could be improved, the rest are questions.
Nice improvements!

File: src/generic/generic-component-site-header/generic-component-site-header.twig (hunk 3 above)
Comment (yhuard) [reply src/generic/generic-component-site-header/generic-component-site-header.twig]: I've not updated component-site-header yet :) just the context here

File: src/generic/generic-component-logo/generic-component-logo.twig (hunk 4 above)
Comment (yhuard) [reply src/generic/generic-component-logo/generic-component-logo.twig]: what do you mean?

File: src/generic/generic-component-site-header/generic-component-site-header.twig (hunk 3 above)
Comment (kalinchernev) [reply src/generic/generic-component-site-header/generic-component-site-header.twig]: :+1:

File: src/generic/generic-component-logo/generic-component-logo.twig (hunk 4 above)
Comment (kalinchernev) [reply src/generic/generic-component-logo/generic-component-logo.twig]: In fact, seems `extra_classes` is documented to have a default, but it's not set as such, maybe it's me getting confused about this specific property?

Comment (yhuard) [reply src/generic/generic-component-logo/generic-component-logo.twig]: looks like you've just found a severe flaw in our normalization process 😀

**Review by kalinchernev — APPROVED** [review]

**Pull request merged**